static/health-check.cast

```diff
@@ -0,0 +1,219 @@
+{"version": 2, "width": 92, "height": 24, "timestamp": 1647631954, "env": {"SHELL": "/run/current-system/sw/bin/fish", "TERM": "xterm-kitty"}}
```

Comment:
Should we have the casts under git-lfs? Although it is human readable it is not really that much human readable.

Reply:
They're not too large file-size-wise too, I think keeping them in the normal tree seems ok. We could disable diffs for them by setting

```
/static/*.cast -diff
```

in the `.gitattributes` so `git diff` will treat them as binaries